core-api-library/library/src/main/java/net/gini/android/core/api/authorization/UserRemoteSource.kt

```diff
@@ -61,11 +62,11 @@ internal class UserRemoteSource(
     private fun basicHeaderMap(): Map<String, String> {
         val encoded = Base64.encodeToString("${clientId}:${clientSecret}".toByteArray(), Base64.NO_WRAP)
         return mapOf("Accept" to "application/json",
```

Comment:
[Suggestion] what about adding another class to `BasicAuthorizatonHeader` for the `mapOf("Accept" to "application/json"` ?

Reply:
I agree, thanks for spotting this.